sql/hive/benchmarks/OrcReadBenchmark-results.txt

```diff
@@ -3,220 +3,220 @@ SQL Single Numeric Column Scan
-Native ORC MR                                      1126           1149          33          0.9        1073.7       1.0X
-Native ORC Vectorized                              1136           1141           7          0.9        1083.4       1.0X
-Hive built-in ORC                                   589            595           8          1.8         561.4       1.9X
+Native ORC MR                                      1227           1236          13          0.9        1169.9       1.0X
+Native ORC Vectorized                               190            233          62          5.5         181.2       6.5X
+Hive built-in ORC                                   882            925          64          1.2         841.5       1.4X
 
 OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
 Single Struct Column Scan with 100 Fields:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 -------------------------------------------------------------------------------------------------------------------------
-Native ORC MR                                       9880           9995         163          0.1        9422.1       1.0X
-Native ORC Vectorized                               9815           9868          75          0.1        9359.9       1.0X
-Hive built-in ORC                                   3292           3382         127          0.3        3139.3       3.0X
+Native ORC MR                                      10839          10916         109          0.1       10337.1       1.0X
+Native ORC Vectorized                               1700           1729          41          0.6        1621.5       6.4X
+Hive built-in ORC                                   6408           6512         148          0.2        6110.8       1.7X
 
 OpenJDK 64-Bit Server VM 1.8.0_312-b07 on Linux 5.11.0-1022-azure
-Intel(R) Xeon(R) Platinum 8272CL CPU @ 2.60GHz
+Intel(R) Xeon(R) CPU E5-2673 v4 @ 2.30GHz
 Single Struct Column Scan with 300 Fields:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 -------------------------------------------------------------------------------------------------------------------------
-Native ORC MR                                      31446          31932         687          0.0       29988.9       1.0X
-Native ORC Vectorized                              31467          31601         191          0.0       30008.9       1.0X
-Hive built-in ORC                                  10835          10879          62          0.1       10333.5       2.9X
+Native ORC MR                                      32949          33157         294          0.0       31422.8       1.0X
+Native ORC Vectorized                              31820          32106         404          0.0       30346.3       1.0X
```

Comment:
Vectorized has no improvement than non-vectorized, because here we read struct with 300 fields, more than [the threshold for whole stage code-gen of max # of fields: 100](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L1368).